controllers/openstackdataplanerole_controller.go

```diff
@@ -227,57 +226,11 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 				err.Error()))
 			return ctrl.Result{}, err
 		}
-		if result.RequeueAfter > 0 {
+		if deployResult != nil {
+			result = *deployResult
 			return result, nil
 		}
 
-		// Call DeployNovaExternalCompute individually for each node
```

Comment:
ack, remove creating the NovaExternalCompute CR from within the controller code and create it instead in deployment.go